resources/providers/awslib/securityhub/securityhub.go

```diff
@@ -0,0 +1,51 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package securityhub
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/securityhub"
+	"github.com/elastic/cloudbeat/resources/fetching"
+)
+
+type (
+	Service interface {
+		Describe(ctx context.Context) (SecurityHub, error)
+	}
+
+	SecurityHub struct {
+		Enabled bool
+		*securityhub.DescribeHubOutput
+	}
+)
+
+func (b SecurityHub) GetResourceArn() string {
+	if b.HubArn == nil {
+		return ""
+	}
+	return *b.HubArn
+}
+
+func (b SecurityHub) GetResourceName() string {
+	return ""
```

Comment:
Let's make the resource name client friendly, I suggest `security hub - <AWS_REGION>`